spec/custom_rules/base_spec.rb

```diff
@@ -44,14 +42,12 @@ def rule_text
         expect(base_rule).to receive(:audit_impl)
           .and_return(%w[r1 r2 r3])
 
-        dontcare = double('cfn_model')
-
         expected_violation = Violation.new(id: 'F3333',
                                            type: Violation::FAILING_VIOLATION,
                                            message: 'This is an epic fail!',
                                            logical_resource_ids: %w[r1 r2 r3])
 
-        expect(base_rule.audit(dontcare)).to eq expected_violation
+        expect(base_rule.audit(nil)).to eq expected_violation
```

Comment:
reason for getting rid of "dontcare"? (which shows more intention)

Reply:
After my changes rspec was complaining about this double leaking into other specs. Since we are not really using the double features at all, nil seems to be an equivalent and the tests pass.

Reply:
that's cool - i'd set a variable named dontcare = nil just to make it clear that the argument has nothing to do with the outcome of the test